recipes/simple-build-system/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set version = "0.0.18" %}
+{% set sha_value = "f8135874cfb599f449363ce11ca45c691bbc7ea4407511c90367cc117dff9f96" %}
+
+#NB get sha256 with: curl -sL <URL> | openssl sha256
+
+package:
+  name: simple-build-system
+  version: {{ version }}
+
+source:
+  url: https://github.com/mctools/simplebuild/archive/v{{ version }}.tar.gz
+  sha256: {{ sha_value }}
```

Please don't use an indirection here
```suggestion
  sha256: f8135874cfb599f449363ce11ca45c691bbc7ea4407511c90367cc117dff9f96
```

---

Ok, done!